d2l-pdf-viewer.js

```diff
@@ -463,7 +463,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-pdf-viewer">
 			fullscreen-available="[[_isFullscreenAvailable]]"
 			is-fullscreen="[[_isFullscreen]]"
 			enable-print="[[enablePrint]]"
-			enable-download="[[enableDownload]]"
+			enable-download="[[_enableDownload]]"
```

Comment:
no need for computed property I think
```suggestion
			enable-download="[[_checkDownloadEnabled(enableDownload, _downloadManager)]]"
```